programs/ssl/ssl_server2.c

```diff
@@ -693,7 +700,7 @@ struct options {
     const char *cid_val_renego; /* the CID to use for incoming messages
                                  * after renegotiation                      */
     int reproducible;           /* make communication reproducible          */
-    uint32_t max_early_data_size; /* max amount of early data               */
+    const char *max_early_data_size; /* max amount list of early data         */
```

Comment:
I am not sure to understand why we need a list of two sizes, especially in this PR.

Reply:
It should be moved to #8532. 

That's to make sure we use the max_early_data_size from session.

Reply:
That test the early data is rejected due to ticket flags.